pyls_jsonrpc/streams.py

```diff
@@ -25,6 +25,10 @@ def listen(self, message_consumer):
 
             if request_str is None:
                 break
+                
+            # to check for windows line endings    
```

Comment:
I don't quite understand why this is necessary here? Doesn't the line `while line and line.strip()` strip all whitespace, including windows line endings? 

What is your rfile where you're seeing this break?